tests/test_zeus_client.py

```diff
@@ -68,23 +81,22 @@ def test_validate_dates(self):
     @mock.patch('zeus.interfaces.rest.requests')
     def test_post_empty_log(self, mock_requests):
         logs = []
+        url = '{}/logs/{}/ZeusTest'.format(FAKE_SERVER, FAKE_TOKEN)
```

Comment:
We need to know the behavior of `urlparse.urljoin` or `posixpath.join` functions, isn't it just adding complexity (and it does not fix the slash problem). We could not expect the behavior when i wrote https://github.com/CiscoZeus/zeus_api/pull/125 for example.

```python
# easy to debug
url = '{}/logs/{}/ZeusTest'.format(FAKE_SERVER, FAKE_TOKEN)

urlparse.urljoin('http://www.fake.com/', posixpath.join('logs', '/token/', '/test'))
# => what will be (without executing this code). 
```

In addition, I've just googled `python client github`, figuring out most of OSS does just `+` or `format`.

- [google-cloud-python](https://github.com/GoogleCloudPlatform/google-cloud-python/blob/master/bigquery/google/cloud/bigquery/client.py#L136)
- [kubernetes-incubator](https://github.com/kubernetes-incubator/client-python/blob/f399ece006d43970a9d25052798782db1407fd05/kubernetes/client/api_client.py#L145)
(mot sure these are just random)

if you still think it's better to use `urljoin`, I'm okay to change.